config/services.yaml

```diff
@@ -40,6 +44,10 @@ services:
             string $fromAddress: "%env(MAILER_FROM_ADDR)%"
             string $fromName: "%env(MAILER_FROM_NAME)%"
 
+    App\Util\MdappLogger:
+        arguments:
+            - '%kernel.project_dir%/var/log/mdapp.log'
```

Comment:
Can use %kernel.logs_dir% to get to var/log/ folder
ex: %kernel.logs_dir%/mdapp.log